packages/components/psammead-caption/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-caption",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Update `package.json` . #446 has been merged 
```suggestion
  "version": "1.1.1",
```